suitescript/suitescript.d.ts

```diff
@@ -1234,7 +1236,7 @@ declare function nlapiGetLineItemDateTimeValue(type:string, fldnam:string, linen
  * @param linenum
  * @param value
  */
-declare function nlapiSetLineItemValue(type:string, fldnam:string, linenum:any, value:string):void;
+declare function nlapiSetLineItemValue(type:string, fldnam:string, linenum:any, value:any):void;
```

Comment:
Much like my last comment, based on history.  But, I'll take your approach and change to string|number